### PR TITLE
Update, cleanup and improve PEtab import

### PR DESCRIPTION
**Pull request by dweindl**
- [x] new petab release
- [x] new amici release (after https://github.com/ICB-DCM/AMICI/pull/921)
- [x] use PEtab observable tables
- [x] use PEtab yaml files

### CODE REVIEW

**Comment by dweindl** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ pypesto/objective/petab_import.py  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=4689415&bid=15678198)

**Comment by dweindl** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ pypesto/objective/petab_import.py  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=4689415&bid=15678198)

**Comment by dweindl** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ pypesto/objective/petab_import.py  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=4689415&bid=15678198)

**Comment by dweindl** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ pypesto/objective/petab_import.py  -5
+ test/test_petab_import.py  -17
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=4689415&bid=15678198)

**Comment by dweindl** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ test/test_petab_import.py  -17
+ pypesto/objective/petab_import.py  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=4689415&bid=15678198)

File: pypesto/objective/petab_import.py [hunk 1]
```diff
@@ -388,16 +287,17 @@ def create_objective(self,
...
-                          ub=self.petab_problem.ub,
-                          x_fixed_indices=self.petab_problem.x_fixed_indices,
-                          x_fixed_vals=self.petab_problem.x_fixed_vals,
-                          x_names=self.petab_problem.x_ids)
+        problem = Problem(
+            objective=objective,
+            lb=self.petab_problem.lb_scaled,
+            ub=self.petab_problem.ub_scaled,
+            x_fixed_indices=self.petab_problem.x_fixed_indices,
+            x_fixed_vals=self.petab_problem.x_nominal_fixed_scaled,
```
Comment (FFroehlich) [inline pypesto/objective/petab_import.py]:
```suggestion
            x_fixed_vals=self.petab_problem.x_fixed_values,
```

Comment (yannikschaelte) [reply pypesto/objective/petab_import.py]: We require the values to be scaled, so if I am not missing something, we need ...scaled... here, right?

Comment (FFroehlich) [reply pypesto/objective/petab_import.py]: `petab_problem.x_nominal_fixed_scaled` doesn't exist anymore

**Comment by dweindl** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ test/test_petab_import.py  -17
+ pypesto/objective/petab_import.py  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=4689415&bid=15678198)

File: pypesto/objective/petab_import.py (hunk 1 above)
Comment (FFroehlich) [reply pypesto/objective/petab_import.py]: oh it does on develop :( my bad ...

**Comment by dweindl** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ test/test_petab_import.py  -17
+ pypesto/objective/petab_import.py  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=4689415&bid=15678198)

**Comment by dweindl** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ pypesto/objective/petab_import.py  -5
+ test/test_petab_import.py  -17
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=4689415&bid=15678198)

**Comment by dweindl** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ pypesto/objective/petab_import.py  -5
+ test/test_petab_import.py  -17
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=4689415&bid=15678198)

**Comment by dweindl** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ pypesto/objective/petab_import.py  -5
+ test/test_petab_import.py  -17
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=4689415&bid=15678198)

**Comment by dweindl** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ test/test_petab_import.py  -17
+ pypesto/objective/petab_import.py  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=4689415&bid=15678198)

**Comment by dweindl** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ pypesto/objective/petab_import.py  -5
+ test/test_petab_import.py  -17
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=4689415&bid=15678198)

**Comment by dweindl** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ pypesto/objective/petab_import.py  -5
+ test/test_petab_import.py  -17
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=4689415&bid=15678198)

**Comment by dweindl** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ test/test_petab_import.py  -17
+ pypesto/objective/petab_import.py  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=4689415&bid=15678198)

**Comment by yannikschaelte** [comment]
This does not yet reconcile with the amici.petab_objective.simulate_petab functions, but at least it makes everything run again (at least for the considered models). @dweindl @FFroehlich wanna have a look?

Next then: Add gradient checks, and merge with the amici functions (not here yet).

**Comment by dweindl** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ test/test_petab_import.py  -17
+ pypesto/objective/petab_import.py  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=4689415&bid=15678198)

**Comment by dweindl** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ test/test_petab_import.py  -17
+ pypesto/objective/petab_import.py  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=4689415&bid=15678198)

**Comment by dweindl** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ pypesto/objective/petab_import.py  -5
+ test/test_petab_import.py  -17
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=4689415&bid=15678198)

**Pull request merged**